ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerDelivery.sol

```diff
@@ -263,21 +257,33 @@ abstract contract WormholeRelayerDelivery is WormholeRelayerBase, IWormholeRelay
             return;
         }
 
-        DeliveryResults memory results = executeInstruction(
-            EvmDeliveryInstruction({
-                sourceChain: vaaInfo.sourceChain,
-                targetAddress: vaaInfo.deliveryInstruction.targetAddress,
-                payload: vaaInfo.deliveryInstruction.payload,
-                gasLimit: vaaInfo.gasLimit,
-                totalReceiverValue: vaaInfo.totalReceiverValue,
-                targetChainRefundPerGasUnused: vaaInfo.targetChainRefundPerGasUnused,
-                senderAddress: vaaInfo.deliveryInstruction.senderAddress,
-                deliveryHash: vaaInfo.deliveryVaaHash,
-                signedVaas: vaaInfo.encodedVMs
-            }
-        ));
+        DeliveryResults memory results;
+
+        // Check replay protection - if so, set status to receiver failure
+        if(getDeliverySuccessState().deliverySuccessBlock[vaaInfo.deliveryVaaHash] != 0) {
```

Comment:
I checked and didn't see an issue, but flagging that it makes me slightly nervous that now we're not checking replay protection until this point in code